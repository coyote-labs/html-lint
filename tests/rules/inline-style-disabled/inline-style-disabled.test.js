const {htmlLint} = require('../../../index');

global.console = {
  error: jest.fn()
};

test.each([
  'tests/rules/inline-style-disabled/no-error.html',
  'tests/rules/inline-style-disabled/error-inline-style-disabled.html',
])(
  'doctype-first %s',
  async(fixture) => {
    try {
      await htmlLint(fixture);
    } catch ({ message }) {
      expect(global.console.error).toMatchSnapshot();
    }
  },
);
