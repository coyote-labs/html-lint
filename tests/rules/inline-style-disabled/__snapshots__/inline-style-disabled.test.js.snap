// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inline-style-disabled tests/rules/inline-style-disabled/error-inline-style-disabled.html 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      "[101m[html-lint] 1 error in tests/rules/inline-style-disabled/error-inline-style-disabled.html[49m",
    ],
    Array [
      "[90minline-style-disabled: Inline styles should be avoided.[39m
[0m [90m 5 | [39m  [33m<[39m[33m/[39m[33mhead[39m[33m>[39m[0m
[0m [90m 6 | [39m  [33m<[39m[33mbody[39m[33m>[39m[0m
[0m[31m[1m>[22m[39m[90m 7 | [39m    [33m<[39m[33mp[39m style[33m=[39m[32m\\"color:red;\\"[39m[33m>[39m[33mHello[39m[33m<[39m[33m/[39m[33mp[39m[33m>[39m[0m
[0m [90m   | [39m       [31m[1m^[22m[39m [31m[1mInline styles should be avoided.[22m[39m[0m
[0m [90m 8 | [39m  [33m<[39m[33m/[39m[33mbody[39m[33m>[39m[0m
[0m [90m 9 | [39m[33m<[39m[33m/[39m[33mhtml[39m[33m>[39m[0m",
      "
",
    ],
    Array [
      "
",
    ],
  ],
  "results": Array [
    Object {
      "type": "return",
      "value": undefined,
    },
    Object {
      "type": "return",
      "value": undefined,
    },
    Object {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
